src/components/home/elements/BurnToken.vue

```diff
@@ -0,0 +1,744 @@
+                    this.amount = BigNumber(jsonObj.amount).dividedBy(VSYS_PRECISION).decimalPlaces(8)
+                }
+                if (jsonObj.hasOwnProperty('invoice')) {
+                    this.attachment = jsonObj.invoice
+                }
+                if (protocol !== PROTOCOL) {
+                    this.paused = false
+                    this.qrErrMsg = 'Invalid QR code protocol.'
+                } else if (api !== API_VERSION) {
+                    this.paused = false
+                    this.qrErrMsg = 'API version mismatch.'
+                } else if (opc !== OPC_ACCOUNT) {
+                    this.paused = false
+                    this.qrErrMsg = 'Wrong operation code in QR code.'
+                } else if (!this.isValidRecipient(this.recipient) || this.recipient === '') {
+                    this.paused = false
+                    this.qrErrMsg = 'Invalid address of recipient.'
+                } else {
+                    this.qrErrMsg = void 0
+                }
+            } catch (e) {
+                if (this.isValidRecipient(decodeString)) {
+                    this.recipient = decodeString
+                } else {
+                    this.recipient = 'please scan QR code of recipient'
+                    this.paused = false
+                }
+            }
+        },
+        getSignature: function(signature) {
+            this.coldSignature = signature
+            this.dataObject.timestamp *= 1e6
+            this.coldPageId++
+        },
+        repaintLocation(location, ctx) {
+            if (location !== null) {
+                const {
+                    topLeftCorner,
+                    topRightCorner,
+                    bottomLeftCorner,
+                    bottomRightCorner
+                } = location
+                ctx.strokeStyle = 'orange' // instead of red
+                ctx.beginPath()
+                ctx.moveTo(topLeftCorner.x, topLeftCorner.y)
+                ctx.lineTo(bottomLeftCorner.x, bottomLeftCorner.y)
+                ctx.lineTo(bottomRightCorner.x, bottomRightCorner.y)
+                ctx.lineTo(topRightCorner.x, topRightCorner.y)
+                ctx.lineTo(topLeftCorner.x, topLeftCorner.y)
+                ctx.closePath()
+                ctx.stroke()
+            }
+        },
+        hideQrScan(tabIndex) {
+            if (tabIndex === 0) {
+                this.resetPage()
+                this.pageId = 1
+            } else {
+                this.resetPage()
+                this.coldPageId = 1
+            }
+            this.scanShow = false
+        },
+        isAmountValid(type) {
+            var amount = this.amount
+            if (BigNumber(amount).isEqualTo(0)) {
+                return void 0
+            }
+            return !BigNumber(amount).isNaN() && !this.isWrongFormat(amount) && !this.isInsufficient() && !this.isNegative(amount) && !this.isTokenInsufficient(amount)
+        },
```

Comment:
this method is `isAmountValid` but `isInsufficient` seems check the VSYS balance